### ISSUE TITLE
Xcode12.5 Crash 

### ISSUE THREAD

**Issue by oke331** [0maru/twitter_login#43]
I updated from Xcode12.4 to Xcode12.5.
And then I tried to login on twitter.

This is my code.
```dart
twitterUser = await _twitterLogin.login();
```
But the application crashed.
 
When I use Xcode11.4, the application doesn't crash.

I got following error.
I think it was caused by `redirectURI`.
But I don't know the solution...

<details>
<summary>error log</summary>

```
021-04-27 16:10:24.816157+0900 Runner[474:15661] [AuthenticationSession] The provided scheme is not valid. A scheme should not include special characters such as ":" or "/".

2021-04-27 16:10:24.817931+0900 Runner[474:15661] *** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'The provided scheme is not valid. A scheme should not include special characters such as ":" or "/".'
```

</details>

I use the version 2.1.4.(I am sorry for not using latest version.)

**Comment by xmkevinchen** [comment]
Getting the same crash after upgrading Xcode to 12.5

**Comment by 0maru** [comment]
Sorry it took so long.
This has been fixed in 3.0.8.
